tutorial/Introduction to phyx.js.md

```diff
@@ -0,0 +1,448 @@
+    <http://example.org/phyx-tutorial#phylogeny0_node0> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.obolibrary.org/obo/CDAO_0000140> .
+    <http://example.org/phyx-tutorial#phylogeny0_node10> <http://ontology.phyloref.org/phyloref.owl#has_Sibling> <http://example.org/phyx-tutorial#phylogeny0_node13> .
+    <http://example.org/phyx-tutorial#phylogeny0_node10> <http://purl.obolibrary.org/obo/CDAO_0000149> <http://example.org/phyx-tutorial#phylogeny0_node11> .
+    <http://example.org/phyx-tutorial#phylogeny0_node10> <http://purl.obolibrary.org/obo/CDAO_0000149> <http://example.org/phyx-tutorial#phylogeny0_node12> .
+    <http://example.org/phyx-tutorial#phylogeny0_node10> <http://purl.obolibrary.org/obo/CDAO_0000179> <http://example.org/phyx-tutorial#phylogeny0_node9> .
+    <http://example.org/phyx-tutorial#phylogeny0_node10> <http://purl.obolibrary.org/obo/CDAO_0000187> _:b50 .
+    <http://example.org/phyx-tutorial#phylogeny0_node10> <http://purl.obolibrary.org/obo/CDAO_00002...
+
+
+
+```javascript
+// We can convert these N-Quads into RDF/Turtle for easier interpretation.
+// We only convert a few N-Quads as a demonstration.
+var N3 = require('n3');
+var { Readable } = require("stream")
+
+nQuads.then(nq => {
+    someNqs = nq.slice(0, 1071)
+
+    var streamParser = new N3.StreamParser(),
+    inputStream = Readable.from([someNqs]),
+    streamWriter = new N3.StreamWriter();
+
+    inputStream.pipe(streamParser);
+    streamParser.pipe(streamWriter);
+    streamWriter.pipe(process.stdout);
+});
+
+undefined;
+```
+
+    <http://example.org/phyx-tutorial#phylogeny0> <http://ontology.phyloref.org/phyloref.owl#newick_expression> "(Parasuchia,(rauisuchians,Aetosauria,(sphenosuchians,(protosuchians,(mesosuchians,(Hylaeochampsa,Aegyptosuchus,Stomatosuchus,(Allodaposuchus,('Gavialis gangeticus',(('Diplocynodon ratelii',('Alligator mississippiensis','Caiman crocodilus')Alligatoridae)Alligatoroidea,('Tomistoma schlegelii',('Osteolaemus tetraspis','Crocodylus niloticus')Crocodylinae)Crocodylidae)Brevirostres)Crocodylia))Eusuchia)Mesoeucrocodylia)Crocodyliformes)Crocodylomorpha));";
+        <http://purl.obolibrary.org/obo/CDAO_0000148> <http://example.org/phyx-tutorial#phylogeny0_node0>;
+        <http://purl.org/dc/terms/source> _:b1_b165;
+        a <http://ontology.phyloref.org/phyloref.owl#ReferencePhylogenyEvidence>;
+        <http://www.w3.org/2000/01/rdf-schema#isDefinedBy> _:b1_b0.
+
+
+## Navigating a Phyx document as a JSON file
+
+Most phyx.js wrappers have been designed to help interpret the more complex parts of a Phyx file, such as the phyloreferences, specifiers, phylogenies, citations and the entire Phyx document. However, since every Phyx document is also a JSON document, we encourage the use of standard JSON libraries to access much of the information in the Phyx document. In some cases, as in the demonstration below, this requires more complex code than is necessary by using the phyx.js wrappers.
```

Comment:
```suggestion
Most phyx.js wrappers have been designed to help interpret the more complex parts of a Phyx file, such as the phyloreferences, specifiers, phylogenies, citations and the entire Phyx document. However, since every Phyx document is also a JSON document, much of the information in the Phyx document can be accessed sufficiently easily using standard JSON libraries. In some cases, as in the demonstration below, this requires more complex code than would be necessary by using the phyx.js wrappers.
```